src/Sarif/CodeGenHints.json

```diff
@@ -19,26 +42,26 @@
       }
     }
   ],
-  "*.Properties" :  [
-      {
-        "kind": "DictionaryHint",
-        "arguments": {
-          "valueTypeName": "SerializedPropertyInfo",
-          "namespaceName": "Microsoft.CodeAnalysis.Sarif.Readers",
-          "comparisonKind": "ObjectEquals",
-          "hashKind": "ScalarReferenceType",
-          "initializationKind": "SimpleAssign"
-        }
-      },
-      {
-        "kind": "PropertyModifiersHint",
-        "arguments": {
-          "modifiers": [
-            "internal",
-            "override"
-          ]
-        }
+  "*.Properties" : [
+    {
+      "kind": "DictionaryHint",
+      "arguments": {
+        "valueTypeName": "SerializedPropertyInfo",
+        "namespaceName": "Microsoft.CodeAnalysis.Sarif.Readers",
+        "comparisonKind": "ObjectEquals",
+        "hashKind": "ScalarReferenceType",
+        "initializationKind": "SimpleAssign"
       }
+    },
+    {
+      "kind": "PropertyModifiersHint",
+      "arguments": {
+        "modifiers": [
+          "internal",
+          "override"
+        ]
+      }
+    }
```

Comment:
(You can't see it in CodeFlow, but Git shows a diff here even though comparing both sides shows them to be the same.)

---
In reply to: [64995043](https://github.com/Microsoft/sarif-sdk/pull/316#discussion_r64995043) [](ancestors = 64995043)